--- source/cloudsc-dwarf/src/cloudsc_gpu/cloudsc_gpu_scc_k_caching_mod.F90.bak	2026-07-13 10:22:19.000000000 +0200
+++ source/cloudsc-dwarf/src/cloudsc_gpu/cloudsc_gpu_scc_k_caching_mod.F90	2026-07-13 10:22:19.000000000 +0200
@@ -2183,7 +2183,7 @@
           ZLAMBDA = ZPOW(REAL(YRECLDP%RCL_FAC1, JPRL) / ((ZRHO*ZPRECLR)), REAL(YRECLDP%RCL_FAC2, JPRL))            ! ZPRECLR=kg/kg
 
           ZEVAP_DENOM = YRECLDP%RCL_CDENOM1*ZESATLIQ - YRECLDP%RCL_CDENOM2*ZTP1(JK_I)*ZESATLIQ + YRECLDP%RCL_CDENOM3*ZPOW(ZTP1( &
-          & JK), 3._JPRL)*PAP(JL, JK)
+          & JK_I), 3._JPRL)*PAP(JL, JK)
 
           ! Temperature dependent conductivity
           ZCORR2 = (ZPOW(ZTP1(JK_I) / 273._JPRL, 1.5_JPRL)*393._JPRL) / (ZTP1(JK_I) + 120._JPRL)
